exporter/loggingexporter/internal/otlptext/databuffer.go

```diff
@@ -258,6 +264,32 @@ func (b *dataBuffer) logLinks(description string, sl ptrace.SpanLinkSlice) {
 	}
 }
 
+func (b *dataBuffer) logExemplars(description string, se pmetric.ExemplarSlice) {
+	if se.Len() == 0 {
+		return
+	}
+
+	b.logEntry("%s:", description)
+
+	for i := 0; i < se.Len(); i++ {
+		e := se.At(i)
+		b.logEntry("Exemplar #%d", i)
+		b.logEntry("     -> Trace ID: %s", e.TraceID())
+		b.logEntry("     -> Span ID: %s", e.SpanID())
+		b.logEntry("     -> Timestamp: %s", e.Timestamp())
+		switch e.ValueType() {
+		case pmetric.ExemplarValueTypeInt:
+			b.logEntry("     -> Value: %d", e.IntValue())
+		case pmetric.ExemplarValueTypeDouble:
+			b.logEntry("     -> Value: %f", e.DoubleValue())
+		}
+		if e.FilteredAttributes().Len() == 0 {
```

Comment:
This check is already part of `logAttributes` no?

Reply:
You right. I took this over from other places we're printing out attribute, but the checks are unnecessary. I'll remove them.